dds/DCPS/RTPS/RtpsDiscovery.h

```diff
@@ -34,771 +35,13 @@ using DCPS::AtomicBool;
 typedef RcHandle<Spdp> ParticipantHandle;
 typedef OPENDDS_MAP_CMP(GUID_t, ParticipantHandle, GUID_tKeyLessThan) ParticipantMap;
 typedef OPENDDS_MAP(DDS::DomainId_t, ParticipantMap) DomainParticipantMap;
+typedef OpenDDS::DCPS::RcHandle<RtpsDiscoveryConfig> RtpsDiscoveryConfig_rch;
```

Comment:
Move to RtpsDiscoveryConfig.h after the class definition so that all users of RtpsDiscoveryConfig have access to this.
Also you can remove the `OpenDDS::` prefix since it's already in that namespace.

Reply:
Updates based on feedback have been commited.